### ISSUE TITLE
Deflake WPT and e2e

### ISSUE THREAD

**Issue by sadym-chromium** [GoogleChromeLabs/chromium-bidi#317]
Sometimes the WPT and e2e tests are flaky. They should be fixed.

* https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/3481576076

**Comment by thiagowfx** [comment]
The following test targets are increasingly flake in the past few days:

```
tests/test_browsing_context.py 
  test_browsingContext_navigateSameDocumentNavigation_waitComplete_navigated
  test_browsingContext_navigateSameDocumentNavigation_waitInteractive_navigated 
  test_browsingContext_navigateSameDocumentNavigation_navigated

tests/test_nested_browsing_context.py 
  test_nestedBrowsingContext_navigateSameDocumentNavigation_navigated
```

**Comment by thiagowfx** [comment]
Example: https://github.com/GoogleChromeLabs/chromium-bidi/actions/runs/4203980102/jobs/7294046160

**Comment by sadym-chromium** [comment]
Consider fixed by #462
